roles/rsyslog/README.md

```diff
@@ -386,20 +385,20 @@ __rsyslog_conf_yourname:
 ```
 Type is for adding prefix to the file name to manage the order of the configuration loaded.  In the viaq case, only .conf files set type 'modules', 'output', and 'template' are set.  By setting 'modules', for instance, prefix "10-" is added to the "name".  I.e., if the name is "mmk8s" and type is "modules", the file name "10-mmk8s.conf" is constructed.  'template' is mapped to '20-'; 'output' is mapped to '30-'.  The digits ensure the configuration files are loaded in the correct order.  The type and prefix mapping is defined in rsyslog_weight_map in ./roles/rsyslog/defaults/main.yml.
 
-If the deploy destination is other than {{ rsyslog_config_dir }}, the path is to be set to path.
+If the deploy destination is other than `/etc/rsyslog.d`, the path is to be set to path.
 
 By default, the generated configuration file starts with a comment "# Ansible managed".  It could break some type of configurations.  For instance, "version=2" must be the first line in a rulebase file.  To avoid having "# Ansible managed", set true to nocomment.
 
 Some full path configuration may be referred from other configuration file, e.g., 20-viaq_formatting.conf refers parse_json.rulebase as follows.
 ```
-action(type="mmnormalize" ruleBase="{{ rsyslog_config_dir }}/parse_json.rulebase" variable="$!MESSAGE")
+action(type="mmnormalize" ruleBase="{{ __rsyslog_config_dir }}/parse_json.rulebase" variable="$!MESSAGE")
```

Comment:
this is possibly different than `/etc/rsyslog.d`?

Reply:
Hmmm, do we want to introduce an extra var like `logging_aux_dir` (or `logging_rulebase_dir` more specifically)? I think we don't want to allow users change `rsyslog_config_dir` (based on the discussion https://github.com/linux-system-roles/logging/pull/131#issuecomment-638100420)?

Reply:
sorry, I don't know what rule base dir is, but does anybody need to set it to anything else than `/etc/rsyslog.d`? @richm is your comment implying that this may be needed?

Reply:
If not, you should replace the template expansion by its value, since this is a documentation file file and who wants to see templates referring to internal variables in user documentation?

Reply:
I thought that one of the purposes of this PR was to address this issue:

> Only indirectly related remark, but is there any reason for having logging_config_dir/rsyslog_config_dir configurable? Does one ever need to change it? Other roles usually have the path to their configuration files hardoded, because that's where the software they manage expects to find it.

I was confused because some of the changes change `rsyslog_config_dir` to `/etc/rsyslog.d`, and some keep `rsyslog_config_dir` or use `__rsyslog_config_dir` instead.

Reply:
Ah, sorry. I think I should have used __rsyslog_config_dir in all the places. Taking a look and fixing them.